### ISSUE TITLE
Restructure to markdown function

### ISSUE THREAD

**Issue by filipefilardi** [nagoya-foundation/r-functions-performance#6]
All informations that we are providing are necessary or just min time, max time, avg time? 

Should we provide another, like "package" or "syntax"? @blmayer

**Comment by blmayer** [comment]
I'd like to see the functions complexity. But that would demand a big leap. I recommend opening a milestone for that.

**Comment by filipefilardi** [comment]
Do you know a way to do it without reading all source code?

**Comment by blmayer** [comment]
Yes, each function should be tested with inputs of different size. For example, the sum receives a vector as input, so testing with vectors of length 1, 10, 100, 1000 ... would give us a good insight of how the function behaves. Also, changing the magnitude of the numbers in the vector would be interesting to check.

**Comment by filipefilardi** [comment]
I think we should define what is our goal.

I mean, the whole point is provide information to improve your performance when you are using this in "real life". That's why we are trying to simulate real life situations with a dataset with "flaws".

If we use vectors with different sizes, i don't think we'll get the core information we need. Performance in small data is irrelevant when we consider huge datasets.

**TL;DR** I think we should put this as second plan
